met-web/src/services/feedbackService/types.ts

```diff
@@ -1,4 +1,5 @@
 import { FeedbackStatusEnum } from 'models/feedback';
+import { string } from 'yup';
```

Comment:
TypeScript already has a `string` type; I don't think this needs to be imported from yup here.